### PR TITLE
AT_05.02_008 | Create a new Pipeline

### PR DESCRIPTION
**Pull request by Nzs1**
https://trello.com/c/AsZlf2Dg/1096-at0502008-create-a-new-pipeline

### CODE REVIEW

**Review by byKosta — APPROVED** [review]
good job!

File: cypress/e2e/newPipelineCreate.cy.js [hunk 1]
```diff
@@ -50,4 +50,14 @@ describe('Create a new Pipeline', () => {
         cy.get('li.jenkins-breadcrumbs__list-item>a[href="/"]').click()
         cy.get('table.jenkins-table').should('contain', 'TestPipeline')
     })
+
+    it('Create a new Pipeline', () => {
+        cy.get('span.task-link-text').contains('New Item').click({force: true})
+        cy.get('input#name').type('testProject')
```
Comment (Rom-T) [inline cypress/e2e/newPipelineCreate.cy.js:56]: Please move `testProject` in the fixtures

File: cypress/e2e/newPipelineCreate.cy.js [hunk 2]
```diff
@@ -50,4 +50,14 @@ describe('Create a new Pipeline', () => {
...
+        cy.get('span.label').contains('Pipeline').click()
```
Comment (Rom-T) [inline cypress/e2e/newPipelineCreate.cy.js:57]: Please move `Pipeline` in the fixtures

File: cypress/e2e/newPipelineCreate.cy.js [hunk 3]
```diff
@@ -50,4 +50,14 @@ describe('Create a new Pipeline', () => {
...
+        cy.get('#ok-button').click()
+        cy.get('button[name="Submit"]').click()
+        cy.get('a.model-link').contains('Dashboard').click()
```
Comment (Rom-T) [inline cypress/e2e/newPipelineCreate.cy.js:60]: Please move `Dashboard` in the fixtures

File: cypress/e2e/newPipelineCreate.cy.js [hunk 4]
```diff
@@ -50,4 +50,14 @@ describe('Create a new Pipeline', () => {
         cy.get('li.jenkins-breadcrumbs__list-item>a[href="/"]').click()
         cy.get('table.jenkins-table').should('contain', 'TestPipeline')
     })
+
+    it('Create a new Pipeline', () => {
```
Comment (Rom-T) [inline cypress/e2e/newPipelineCreate.cy.js:54]: You need to add the name of the AT card

**Pull request closed without merging**